tap_asana/streams/sections.py

```diff
@@ -1,14 +1,23 @@
 
 from singer import utils
 from tap_asana.context import Context
-from tap_asana.streams.base import Stream, asana_error_handling
+from tap_asana.streams.base import Stream, asana_error_handling, REQUEST_TIMEOUT
 
 @asana_error_handling
 def get_sections_for_projects(project_gid, owner, opt_fields):
+  # Set request timeout to config param `request_timeout` value.
```

Comment:
@harshpatel4crest This code snippet is common among all the files. Can't you implement it in a common file and assign value to `request_timeout` by calling that function.
I can see that duplicate unit test cases are written because of this redundant code snippet for different files

Reply:
Removed the common code and placed it in the `base.py`.